### PR TITLE
Make command usage in help for required arguments consistent

### PR DESCRIPTION
**Pull request by Jackenmen**
### Type

- [ ] Bugfix
- [x] Enhancement
- [ ] New feature

### CODE REVIEW

File: redbot/core/core_commands.py [hunk 1]
```diff
@@ -1925,7 +1919,7 @@ async def status(self, ctx: commands.Context, *, status: str):
             await ctx.bot.change_presence(status=status, activity=game)
             await ctx.send(_("Status changed to {}.").format(status))
 
-    @_set.command(name="streaming", aliases=["stream"])
+    @_set.command(name="streaming", aliases=["stream"], usage="[(<streamer> <stream_title>)]")
```
Comment (bobloy) [inline redbot/core/core_commands.py:1941]: While weird, I like this method.

**Review by bobloy — DISMISSED** [review]
All your changes look good to me.

There were a few you missed. Would you like me to list them here, or make a separate PR?

**Comment by bobloy** [comment]
I'll just list them since it's not as many as I thought. I was confused about `require_var_positional`'s interaction with `Greedy`

- Image:
  - `gif`
  - `gifr`
- Permissions:
  - `permissions canrun`
- Core:
  - `set api remove`

Editing to add more as I find them.

**Comment by Jackenmen** [comment]
I used `require_var_positional=True` in `[p]set api remove` and edited `[p]gif` and `[p]gifr` to be a consume all argument as it didn't make much sense to begin with.
`[p]permissions canrun` wasn't using var positional argument (`*args`), so I haven't touched that one.
Thanks for the help @bobloy!

File: redbot/cogs/permissions/permissions.py [hunk 2]
```diff
@@ -393,9 +391,6 @@ async def permissions_addglobalrule(
 
         `<who_or_what>` is one or more users, channels or roles the rule is for.
```
Comment (Flame442) [inline redbot/cogs/permissions/permissions.py]: Should this one be `<who_or_what...>`?

File: redbot/cogs/permissions/permissions.py [hunk 3]
```diff
@@ -428,9 +421,6 @@ async def permissions_addguildrule(
...
```
Comment (Flame442) [inline redbot/cogs/permissions/permissions.py]: Should this one be `<who_or_what...>`?

File: redbot/cogs/permissions/permissions.py [hunk 4]
```diff
@@ -455,19 +445,14 @@ async def permissions_removeglobalrule(
...
```
Comment (Flame442) [inline redbot/cogs/permissions/permissions.py]: Should this one be `<who_or_what...>`?

File: redbot/cogs/permissions/permissions.py [hunk 5]
```diff
@@ -482,9 +467,6 @@ async def permissions_removeguildrule(
...
```
Comment (Flame442) [inline redbot/cogs/permissions/permissions.py]: Should this one be `<who_or_what...>`?

File: redbot/core/core_commands.py [hunk 6]
```diff
@@ -1431,12 +1427,10 @@ async def unload(self, ctx: commands.Context, *cogs: str):
             for page in pagify(total_message):
                 await ctx.send(page)
 
-    @commands.command(name="reload")
+    @commands.command(name="reload", require_var_positional=True)
```
Comment (Flame442) [inline redbot/core/core_commands.py]: Any reason why this `name` is here when it is the same as the function name?

**Review by Flame442 — CHANGES_REQUESTED** [review]

File: redbot/core/core_commands.py (hunk 6 above)
Comment (Jackenmen) [reply redbot/core/core_commands.py]: It was like this already so I just haven't touched it. Just changed it now.

**Review by Flame442 — APPROVED** [review]

**Pull request merged**